src/Util/Emailer.php

```diff
@@ -134,6 +134,18 @@ public function send(IEmail $email)
 			return true;
 		}
 
+		// @see https://github.com/friendica/friendica/issues/9142
+		$countMessageId = 0;
+		foreach ($email->getAdditionalMailHeader() as $name => $value) {
+			if (strtolower($name) == 'message-id') {
+				$countMessageId += count($value);
+			}
+		}
+		if ($countMessageId > 0) {
+			$this->logger->warning('More than one Message-ID found - RFC violation', ['email' => $email]);
+			return false;
```

Comment:
No no no no no, we can't just discard the email because we failed to build it properly. This isn't related to any user error so it should not prevent an email send.

Reply:
hmkay .. I thought as a RFC violation, it isn't trustable anymore .. and in the worst case, Mailservers suspects just mails and deny them (afaik mailservers are really restrictive in such cases because of spam ..)

Reply:
Yes but the solution isn't to silently discard the emails even before they can be denied by remote mail servers. It is a code issue we will fix that doesn't warrant this stopgap.

Reply:
OK, I changed it back for now. At a later stage, we should think about a proper solution .. Like cleanup such cases and log them ...

Reply:
You already added logging which is great, now we can move on to the proper solution.